app/views/pages/developer-guide/platformos-workflow/effective-development-using-pos-cli.liquid

```diff
@@ -0,0 +1,185 @@
+  "start": "npx concurrently npm:clean npm:lr npm:dev-*"
+}
+```
+
+Read more about [npm scripts](https://docs.npmjs.com/misc/scripts).
+
+We are using webpack for assets bundling, livereload, and concurrently to run multiple tasks in one terminal.
+
+Having all those tasks defined one by one allows you to run them one by one when you need it:
+
+```shell
+$ npm run clean
+```
+
+{% include 'alert/note', content: 'Keep in mind that command `rm -rf` does not exist on windows, so if you are using windows, change that command, or use [rimraf](https://www.npmjs.com/package/rimraf) for cross-platform compability.' %}
+
+## npx
+
+`npx` is a tool that allows you to run npm packages that expose executable files (binaries). If the package is not installed on your system (locally or globally), it will install it and then execute.
+
+You can run any npm package, without knowing if its installed or not, just by prefixing it with `npx`. For example, we could rewrite our clean task to be cross platform like that:
+
+```shell
+$ npx rimraf app/assets/*
+```
+
+Read more about [npx](https://www.npmjs.com/package/npx).
+
+## livereload
+
+Livereload is broadcasting a "refresh" message when files that you defined are changed.
+
+Let's explain what is happening in `lr` task from example above.
+
+```shell
+npx livereload \"app\" -e \"liquid,graphql,js,css\" -w 1000
+```
+
+1. Run `livereload` via `npx`
+2. Point it to app directory because there located output files that `pos-cli` is syncing
+3. Limit observer to only look for certain file extensions. You might want to add images extensions as an excercise
+4. Wait one second after the change has been detected, then broadcast "refresh" message. This prevents refreshing the page before files have been processed by webpack and synced over to platformOS.
+
+Note the `\` characters before double-quotes. They are needed because npm tasks are in JSON format, which means they use double-quotes to separate name from command to run, so if we did not escape them, it would be a syntax error.
+
+Read more about [livereload](https://www.npmjs.com/package/livereload).
+
+After you have your livereload server running, you need a client.
+Install extension and when you are on a page that you want to be refreshed when files change, click livereload extension icon.
+
+[Chrome extension](https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei)
+[Firefox extension](https://addons.mozilla.org/en-US/firefox/addon/livereload-web-extension/)
+
+Look for icon that looks similar to <img src="https://rawcdn.githack.com/livereload/livereload-extensions/0a8edb9cc7008249c23718386e19d99230e81179/Chrome/LiveReload/IconDisabled@2x.png" width="32" height="32" alt="Livereload icon">.
+
+## concurrently
+
+Finally, when we have all our tasks prepared, we need to run them all in one terminal, to avoid remembering long commands and occupying bunch of terminal windows.
```

Comment:
```suggestion
Finally, when we have all our tasks prepared, we need to run them all at once in one terminal, to avoid remembering long commands and occupying bunch of terminal windows.
```